docs/overview.rst

```diff
@@ -255,58 +258,96 @@ information about the cluster. This includes:
 
-* **Subscription**: The name of the subscription itself, with a link to the
-  :ref:`Subscriptions tab of the Organization page<overview-org-overview>`.
 
 For more information on the CrateDB concepts used here, refer to the `CrateDB
 architecture documentation`_ or the :ref:`glossary<glossary>`.
 
+.. _overview-cluster-overview-access:
 
-.. _overview-cluster-overview-metrics:
+Access
+------
+
+The access tab contains your credentials, security settings, allowlist, and
+information about private endpoints.
 
-Configuration, Usage, and Metrics
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. image:: _assets/img/cluster-access.png
+   :alt: Cloud Console cluster access tab
 
-On the rest of the page you will find various cluster metrics. In the center
-you will see an overview of general usage in the past week, including storage,
-memory, and CPU usage. A bar at the top indicates the 85% mark: if your CPU or
-storage usage is above this percentage of the cluster capacity, you should
-consider upgrading by `scaling the cluster`_.
+.. _overview-cluster-overview-credentials:
 
-On the right you will see more immediate metrics, collected from the last hour
-of cluster operations. This includes average queries, average query response
-time, and disk, memory, and CPU usage for the last hour.
+Credentials
+~~~~~~~~~~~
 
-The queries and query response time of the cluster can be seen in more detail
-on the Cluster Metrics page, for which see below. This page can also be
-accessed by clicking the *View metrics* link at the top right.
+Here you can change your CrateDB Console password.
 
+.. _overview-cluster-overview-security:
+
+Security
+~~~~~~~~
+
+Click the box labelled *Deletion Protected* to add deletion protection to your
+cluster. This will ensure that it can only be deleted in either the CrateDB
+Console or croud by a user with :ref:`org admin rights <org-roles>` (or by a
+Crate.io employee superuser).
+
+.. _overview-cluster-overview-allowlist:
+
+Allowlist
+~~~~~~~~~
+
+By using the IP allowlisting feature, you can restrict access to your cluster
+to an indicated IP address or `CIDR block`_. Click the blue *Add
+Address* button and you can fill out an IP address or range and give it a
+meaningful description. Click *Save* to store it or the bin icon to delete a
+range. Keep in mind that once IP allowlisting has been set, you cannot access
+the Admin UI for that cluster from any other address.
+
+If no allowlist address or address range is set, the cluster is publicly
+accessible by default. (Of course, the normal authentication procedures are
+always required.) Only an :ref:`org admin <org-roles>` can change the
+allowlist.
+
+.. _overview-cluster-overview-private-endpoints:
+
+Private endpoints
+~~~~~~~~~~~~~~~~~
+
+A private endpoint, or private link, is a mechanism that allows a secure, 
+private connection to your cluster. Effectively, it allows you to bypass the
+public internet when accessing the environment where your cluster is deployed.
```

Comment:
Maybe worth mentioning that it cannot be used across providers - i.e. from your Azure account to your AWS cluster.